src/ProcessManagerBundle/Resources/config/pimcore/routing.yml

```diff
@@ -16,4 +16,9 @@ process_manager_admin_processes:
   type: coreshop.resources
   resource: |
     alias: process_manager.process
-    only: ['list', 'delete']
\ No newline at end of file
+    only: ['list', 'delete']
+
+process_manager_admin_processes_report:
```

Comment:
don't use annotations for routing please

Reply:
I'm not sure how to change it. Used here routing syntax is not clear for me.